libs/libcoap/patches/0001-manpage.patch

```diff
@@ -0,0 +1,21 @@
+--- a/examples/Makefile.am
++++ b/examples/Makefile.am
+@@ -23,18 +23,6 @@
+ coap_rd_SOURCES = coap-rd.c
+ coap_rd_LDADD = @top_builddir@/.libs/libcoap-$(LIBCOAP_API_VERSION).la
+ 
+-# building the manpages
```

Comment:
Man, I thought this was wrong, but it seems --disable-documentation is just busted upstream, seems like it only disables the checks on building them.  That's vile, please file a bug upstream for that.